v3/integration/lints/lints/not_commiting_genTestCerts.go

```diff
@@ -0,0 +1,58 @@
+package lints
+
+/*
+ * ZLint Copyright 2021 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import (
+	"crypto/sha256"
+	"fmt"
+	"go/ast"
+	"io/ioutil"
+	"strings"
+
+	"github.com/zmap/zlint/v3/integration/lints/lint"
+)
+
+const want = `a132b2f256dd0b68bff043568f0e9bae69089082805da64720168294f835e197`
+
+type NotCommitingGenTestCerts struct{}
```

Comment:
Shouldn't it be "NotCommittingGenTestCerts" with the extra 't' in committing?

Reply:
I am a native English speaker who finds great joy in our bizarre little Frankenstein of a language. But I swear that "committing" gets me every time.